utils/node_local_test.py

```diff
@@ -1294,7 +1294,7 @@ def test_alloc_fail(conf):
 
     pools = get_pool_list()
 
-    if len(pools) > 1:
+    if len(pools) > 0:
```

Comment:
In order for this to be consistent when it's run locally with no pools and via the 'fi' mode it probably wants this loop, as copied from other tests above.

    while len(pools) < 1:
        pools = make_pool(server)

I'm keen to see the results however.